src/base/bittorrent/session.cpp

```diff
@@ -3841,38 +3844,44 @@ bool Session::addMoveTorrentStorageJob(TorrentHandleImpl *torrent, const QString
 
 void Session::moveTorrentStorage(const MoveStorageJob &job) const
 {
-    qDebug("Moving torrent storage to \"%s\"...", qUtf8Printable(job.path));
+    const InfoHash infoHash = job.torrentHandle.info_hash();
+    const TorrentHandleImpl *torrent = m_torrents.value(infoHash);
+    const QString torrentName = (torrent ? torrent->name() : QString {infoHash});
+    LogMsg(tr("\"%1\" is moving storage to \"%2\"...").arg(torrentName, job.path));
```

Comment:
Does it make sense to put a currently ongoing process in the log? If it's for debugging purposes ignore me.